config/config.go

```diff
@@ -358,6 +358,16 @@ func InitConfig() {
 		}
 		SetLogging(level)
 	}
+
+	logLocation := param.Logging_File.GetString()
+	if logLocation != "" {
+		// Note: do not need to close the file, logrus does it for us
+		f, err := os.OpenFile(logLocation, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
```

Comment:
Two things to look into:
- How does this react if I feed it a location that the user running a Pelican command doesn't have permissions to write to?
- Assume I have an empty directory `/test`. If I run Pelican with the flag `-l /test/logs/my_log`, this will fail because we don't handle creating the directory. Check out `os.MkdirAll` elsewhere in `config.go`.